pkg/ds/farm.go

```diff
@@ -263,10 +263,8 @@ func (dsf *Farm) handleDSChanges(changes dsstore.WatchedDaemonSets, quitCh <-cha
 	}
 
 	if len(changes.Created) > 0 {
-		dsf.logger.Infof("The following daemon sets have been created:")
+		dsf.logger.Infof("The following %d daemon sets have been created: %s", len(changes.Created), dsIDs(changes.Created))
 		for _, dsFields := range changes.Created {
-			dsf.logger.Infof("%v", *dsFields)
```

Comment:
well, note that we're now showing just the IDs, not all the fields. If you think you'd rather like to see all the fields instead, let's talk.

Or maybe I can just keep this line. Prepend "created"?